renovate.json

```diff
@@ -33,6 +33,13 @@
 		"npm"
 	],
 	"packageRules": [
+		{
+			"matchUpdateTypes": ["minor", "patch"],
+			"matchCurrentVersion": "!/^0/",
```

Comment:
Why exclude certain versions?

Reply:
0.x is typically not considered production ready, nor is it clearly defined how semver works before 1.x

let's review those manually

same with major bumps as they are likely coming with breaking changes

Reply:
I can also drop that. I copied the config from the sentry app. And the sentry app copied from text IIRC :see_no_evil:

Reply:
Fine by me. Many libraries use 0.*.* to mark not-production-ready versions.